Gemfile.lock

```diff
@@ -20,6 +20,7 @@ GEM
     minitest-rg (5.1.0)
       minitest (~> 5.0)
     rake (10.3.1)
+    scrub_rb (1.0.1)
```

Comment:
scrub_rb is present in the gemfile I'm on ruby 2.3.1p112 (2016-04-26 revision 54768) [x86_64-darwin14]  This is in the gemfile I did update my bundler bug it still shows up.